src/ansys/mapdl/core/mapdl_geometry.py

```diff
@@ -120,62 +157,428 @@ def _load_iges(self):
+
+        Return the keypoints coordinates as a numpy array:
+
+        >>> keypoints = mapdl.geometry.get_keypoints(return_as_array=True)
+        array([[ 0.00000000e+00,  1.77800000e-02, -3.18000000e-03],
+               [ 0.00000000e+00, -7.62000000e-03, -3.18000000e-03],
+               [ 1.58750000e-02,  1.77800000e-02, -3.18000000e-03],
+               [ 1.58750000e-02, -7.62000000e-03, -3.18000000e-03],
+               [ 0.00000000e+00, -7.62000000e-03,  0.00000000e+00],
+        ...
+
+        When returning an array, you can also choose to output the keypoint
+        ids as the first column:
+
+        >>> keypoints = mapdl.geometry.get_keypoints(return_ids_in_array=True)
+        array([[ 1.00000000e+00,  0.00000000e+00,  1.77800000e-02,
+                -3.18000000e-03],
+               [ 2.00000000e+00,  0.00000000e+00, -7.62000000e-03,
+                -3.18000000e-03],
+               [ 3.00000000e+00,  1.58750000e-02,  1.77800000e-02,
+                -3.18000000e-03],
+               [ 4.00000000e+00,  1.58750000e-02, -7.62000000e-03,
+                -3.18000000e-03],
+               [ 5.00000000e+00,  0.00000000e+00, -7.62000000e-03,
+                 0.00000000e+00],
+               [ 6.00000000e+00,  1.58750000e-02, -7.62000000e-03,
+        ...
+
+        """
+        if return_as_array or return_ids_in_array:
+            if return_ids_in_array:
+                keyp = np.array(self._keypoints[0])
+                keyp_num = np.array(self._keypoints[1])
+                return np.hstack((keyp_num.reshape((-1, 1)), keyp))
+            else:
+                return np.array(self._keypoints[0])
+
+        keypoints, kp_num = self._keypoints
+
+        if return_as_list:
+            keypoints_ = []
+
+            for each_point, each_id in zip(keypoints, kp_num):
+                keypoints_ps = pv.PolyData([each_point])
+                keypoints_ps["entity_num"] = np.array(each_id).reshape((1))
+                keypoints_.append(keypoints_ps)
+
+            return keypoints_
+
+        else:
+            keypoints_pd = pv.PolyData(keypoints)
+            keypoints_pd["entity_num"] = kp_num
+            return keypoints_pd
 
     @property
-    def _lines(self):
+    def _lines(self) -> List[pv.PolyData]:
         """Returns lines cache"""
         if self._lines_cache is None:
             self._lines_cache = self._load_lines()
         return self._lines_cache
 
     @property
-    def lines(self):
-        """Active lines as a pyvista.PolyData"""
-        return self._lines
+    def lines(self) -> pv.MultiBlock:
+        """Obtain the lines geometry
```

Comment:
```suggestion
        """Lines geometry.
```
Alex told me that properties should have a verb in the description. It's basically a "noun." Here, the plural lines is confusing because it should likely be possessive "Lines' geometry," which looks awkward, or better yet "Geometry of the lines."

Reply:
@germa89 Not sure what we are obtaining. I assumed that we get one geometry having several lines. If we want use "Geometry lines", then what do we get back? Is it a line's geometry (geometry of one line) or lines' geometry (geometry of several lines) or are we getting back multiple geometries of multiple lines? Do you see why I don't know what to suggest?!

Reply:
I seee.... we get one geometrical object which contains the geometry of several lines...

Reply:
So I guess we should go for ``Lines geometry``

Reply:
Whatever, what you say it is good. :)